conf/bblayers.conf.sample

```diff
@@ -1,20 +1,22 @@
 # POKY_BBLAYERS_CONF_VERSION is increased each time build/conf/bblayers.conf
 # changes incompatibly
-POKY_BBLAYERS_CONF_VERSION = "2"
+POKY_BBLAYERS_CONF_VERSION = "3"
+
+OEROOT := "${@os.path.abspath(os.path.dirname(d.getVar('FILE', True)))}/../.."
 
 BBPATH = "${TOPDIR}"
 BBFILES ?= ""
 
 BBLAYERS ?= " \
-    ##OEROOT##/meta \
-    ##OEROOT##/meta-poky \
-    ##OEROOT##/meta-openembedded/meta-oe \
-    ##OEROOT##/meta-openembedded/meta-filesystems \
-    ##OEROOT##/meta-openembedded/meta-networking \
-    ##OEROOT##/meta-openembedded/meta-perl \
-    ##OEROOT##/meta-openembedded/meta-python \
-    ##OEROOT##/meta-raspberrypi \
-    ##OEROOT##/meta-security \
-    ##OEROOT##/meta-pelion-edge \
-    ##OEROOT##/meta-nodejs \
+    ${OEROOT}/meta \
```

Comment:
This change doesn't seem right.  Did Yocto change the way it parses the sample files?  See here: https://www.yoctoproject.org/docs/3.1.1/ref-manual/ref-manual.html#structure-build-conf-bblayers.conf

Reply:
Ya I reverted the way it is used in 1.0.0. Raising a PR to bring this change.